### ISSUE TITLE
Font too large, How can I adjust or Reset font size

### ISSUE THREAD

**Issue by irakandjii** [hyOzd/serialplot#62]
Application was working perfectly.  I loaded SerialPlot from the taskbar, Windows 11 Pro for Workstations.
The font used in the:
1) Port selection box,
2) Legend,
3) all of the information UI, (except the plot data selection dropdown box)
Is now a very large font size about ~30 in normal Windows.  The actual font type is unchanged,
![Screenshot 2024-01-17 120630](https://github.com/hyOzd/serialplot/assets/3685515/82b01b91-1536-40ab-a164-a81f79206307)

Is there a config file or other location where I can adjust this?
Version:
[Debug] SerialPlot 0.12.0
[Debug] Revision 7c0005be1c54

**Comment by hyOzd** [comment]
> Is there a config file or other location where I can adjust this?

No, serialplot doesn't have any font settings. This seems like an issue from Qt sdk. Maybe building with a newer version of the Qt sdk would fix the issue.
